zipkin-storage/cassandra-v1/src/main/java/zipkin2/storage/cassandra/v1/CassandraUtil.java

```diff
@@ -142,4 +144,13 @@ public String toString() {
       return "SortTraceIdsByDescTimestamp";
     }
   }
+
+  @SuppressWarnings("JdkObsolete")
+  static List<Date> getDays(long endTs, @Nullable Long lookback) {
+    List<Date> result = new ArrayList<>();
+    for (Long epochMillis : DateUtil.epochDays(endTs, lookback)) {
```

Comment:
Accidental box?

Reply:
well it is a List<Long> but sure no prob making it eagerly unboxed.